apps/api/src/app/events/usecases/create-notification-jobs/create-notification-jobs.usecase.ts

```diff
@@ -142,4 +133,16 @@ export class CreateNotificationJobs {
 
     return steps;
   }
+
+  private calculateExpireAt(command: CreateNotificationJobsCommand) {
```

Comment:
I think it will be a good idea to make this one public or external function so we could add unit tests to it.